### ISSUE TITLE
Does not work when app is on the denylist

### ISSUE THREAD

**Issue by pozhiloy-enotik** [Android1500/GpsSetter#97]
Applications on the denylist may be the primary purpose for using the app, but suddenly the app can't be used on them

**Comment by 7haveeen** [comment]
Yes and the other issue why it takes long to read the location when this app is on.

**Comment by devingDev** [comment]
any fix on this?
one could try to patch an app manually to not detect root but thats quite tedious and instead it would make more sense if it just "replaced" all gps functions completely somehow. (no spoofing dev setting etc, just straight up delivering the gps location you set and only that to all apps)

**Comment by AlawiYNWA** [comment]
same issue here 
can't spoof location when some app in deny list
how to fix it
